code/modules/food_and_drink/condiments.dm

```diff
@@ -46,6 +46,17 @@
 	desc = "Made with real artificial maple syrup!"
 	icon_state = "syrup"
 
+/obj/item/reagent_containers/food/snacks/condiment/soysauce
+	name = "soy sauce"
+	desc = "A dark brown sauce brewed from soybeans and wheat. Salty!"
+	icon_state = "soy-sauce"
+	initial_volume = 30
+	initial_reagents = "soysauce"
+
+	heal(var/mob/M)
+		boutput(M, "<span class='alert'>FUCK, SALTY!</span>")
```

Comment:
```suggestion
	heal(var/mob/M)
		. = ..()	
		boutput(M, "<span class='alert'>FUCK, SALTY!</span>")
```
Should call the parent. `. = ..()` because i vaguely remember the return value of `heal` meaning something? could be wrong though.